diff --git a/cmd/fscrypt/protector.go b/cmd/fscrypt/protector.go
index 32ba4ab..30dc03d 100644
--- a/cmd/fscrypt/protector.go
+++ b/cmd/fscrypt/protector.go
@@ -43,15 +43,6 @@ func createProtectorFromContext(ctx *actions.Context) (*actions.Protector, error
 	}
 	log.Printf("using name: %s", name)
 
-	// We only want to create new login protectors on the root filesystem.
-	// So we make a new context if necessary.
-	if ctx.Config.Source == metadata.SourceType_pam_passphrase && ctx.Mount.Path != "/" {
-		log.Printf("creating login protector on %q instead of %q", "/", ctx.Mount.Path)
-		if ctx, err = modifiedContext(ctx); err != nil {
-			return nil, err
-		}
-	}
-
 	return actions.CreateProtector(ctx, name, createKeyFn)
 }
 
